understanding/22/accessible-authentication-enhanced.html

```diff
@@ -12,6 +12,16 @@ <h1>Understanding Accessible Authentication (Enhanced)</h1>
         <h2>Status</h2>
         <p>This understanding document is part of the <a href="https://w3c.github.io/wcag/guidelines/22/"><strong>draft</strong> WCAG 2.2 content</a>. It may change or be removed before the final WCAG 2.2 is published.</p>
     </section> 
+   
+    <section id="brief">
+        <h2>In brief</h2>
+        <dl>
+            <dt>Goal</dt><dd>Make logins possible with less mental effort.</dd>
+            <dt>What to do</dt><dd>Don’t make people recognize objects or user-supplied non-text information to login.</dd>
```

Comment:
Since we are saying "non-text information" in the Why as well now, I wonder if we should swap it out here?
```suggestion
            <dt>What to do</dt><dd>Don’t make people recognize objects or user-supplied images and media to login.</dd>
```

Reply:
I've been toying with trying to make this _somehow_ reference the minimum, but it is VERY hard to do unless one assumes a user is a aware there is a minimum version, in which case, they theoretically already understand this is an additional requirement...
```suggestion
            <dt>What to do</dt><dd>In addition to the minimum requirements, don’t make people recognize objects or user-supplied images and media to login.</dd>
```

Reply:
(also wondering about "user-supplied"... )

Reply:
> Were we going to replace "transcribe" -> "Type"?

that doesn't have the same connotation though. Transcribe is "read it in one place, then type it up in another place". Otherwise, just having "type" there might give the wrong impression that this is about just having autofill / QR codes / etc that don't require actual typing, which is not the point

Reply:
in the minimum, which was more concentrated, I've adjusted the 'transcribe' wording, and am fairly satisfied with this.
I do not think there's an elegant or easy way to crack the progression of these 2 criteria in a way that is always a 'win'.  My inclination is to stick pretty closely to the approved language for now, until the time that someone comes up with something that everyone responds YES!